<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="sliceSlider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>461</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="framet1ce">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="labelImagen">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loadFile">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar archivo t1ce</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRuta">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frameseg">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>401</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="loadFile2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar archivo seg</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelRuta2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelImagen2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxShowMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>AplicarFIltros</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
